neurom/features/morphology.py

```diff
@@ -176,20 +223,83 @@ def trunk_angles(morph, neurite_type=NeuriteType.all):
     if len(vectors) == 0:
         return []
 
-    def _sort_angle(p1, p2):
-        """Angle between p1-p2 to sort vectors."""
-        ang1 = np.arctan2(*p1[::-1])
-        ang2 = np.arctan2(*p2[::-1])
-        return ang1 - ang2
+    if sort_along is not None:
```

Comment:
for consistency with the below condition `if coords_only`
```suggestion
    if sort_along:
```

Reply:
done